paimon-flink/paimon-flink-common/src/main/java/org/apache/paimon/flink/sink/AsyncLookupSinkWrite.java

```diff
@@ -34,7 +34,7 @@
 
 /**
  * {@link StoreSinkWrite} for tables with lookup changelog producer and {@link
- * org.apache.paimon.CoreOptions#CHANGELOG_PRODUCER_LOOKUP_WAIT} set to false.
+ * org.apache.paimon.CoreOptions#LOOKUP_WAIT} set to false.
  */
 public class AsyncLookupSinkWrite extends StoreSinkWriteImpl {
```

Comment:
You don't need to modify Flink lookup join? Can you explain which specific case has changed after introducing `force-lookup`?

Reply:
I plan to put the modification of `FileStoreLookupFunction` into the next issue. This pr mainly introduces `force-lookup` to change compaction. 

The next pr will modify the validation of `streamingReadingValidate` and modify `FullCacheLookupTable` to implement streaming consumption data. There is almost no need to modify `PrimaryKeyPartialLookupTable`.

Reply:
OK